tests/testthat/test_MsBackendDataFrame.R

```diff
@@ -505,7 +505,7 @@ test_that("spectraData, spectraData<-, MsBackendDataFrame works", {
 
 test_that("show,MsBackendDataFrame works", {
     be <- MsBackendDataFrame()
-    show(be)
+    expect_output(show(be), "MsBackendDataFrame")
```

Comment:
:+1: for testing `show` but aren't the following two lines useless?

Reply:
I will remove them.